### DISCUSSION TITLE
Stopped working on vite dev after version 11

### DISCUSSION THREAD

**Discussion by Luciden** [sapphi-red/vite-plugin-static-copy#41] [Q&A]
We use a custom middleware to intercept json mock files for testing. from version 10-11 the middleware no long intercepts the files as if their path has changed although the "copy" does work and you can pull them via the url: localhost.../static/file.json. Any ideas are appreciated.

-- snippets of config
  base: '',
  build: {
    outDir: path.resolve(__dirname, '../wwwroot/dist/'),
    emptyOutDir: true,
    assetsDir: 'static',
  },
  define: {
    'process.env': process.env,
  },
  plugins: [
    serverPlugin({}),
    !isProd ? viteStaticCopy({
      targets: [
        {
          src: `./mocks/data/*`,
          dest: `/static`

**Comment by sapphi-red** [comment]
I'm not sure without a reproduction. But I guess that that `serverPlugin` handles all the request and doesn't pass any requests to vite-plugin-static-copy.
